<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfWindowClass</class>
 <widget class="QWidget" name="ProfWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EXR</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="PlotWidget" name="plotWidget" native="true"/>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>View</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="accumThreadsCB">
             <property name="text">
              <string>Accumulate Threads</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="threadListWidget">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTreeView" name="treeView"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>widgets/PlotWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>accumThreadsCB</sender>
   <signal>clicked(bool)</signal>
   <receiver>threadListWidget</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
